lg02-pkt-fwd/src/lg02_single_rx_tx.c

```diff
@@ -298,13 +301,17 @@ int main(int argc, char *argv[])
         static unsigned long next = 1, count_ok = 0, count_err = 0;
         int i, data_size;
 
+	// Ensure that randomly generated channel file names are different for device 1 vs device 2.
+        // device '49' vs device '50' since read as ASCII, see above
+	next += (device - 49); 
+
         rxlora(loradev, RXMODE_SCAN);
 
         if (strlen(filepath) > 0) 
             fp = fopen(filepath, "w+");
 
         MSG("\nListening at SF%i on %.6lf Mhz. port%i\n", loradev->sf, (double)(loradev->freq)/1000000, loradev->spiport);
-        fprintf(stdout, "REC_OK: %d,    CRCERR: %d\n", count_ok, count_err);
+        fprintf(stdout, "REC_OK: %d,    CRCERR: %d, next channel RNG: %ld\n", count_ok, count_err, (unsigned)(next/65536) % 32768);
```

Comment:
Maybe add a comment explaining why this (next/65536) % 32768. Also please verify this works because otherwise we might get unexplainable errors after the first 65536 messages.